config/manager/manager.yaml

```diff
@@ -62,6 +62,10 @@ spec:
             value: /etc/aws-credentials/credentials
           - name: RELATED_IMAGE_CONTROLLER
             value: docker.io/amazon/aws-alb-ingress-controller:v2.4.1
+          - name: TARGET_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.annotations['olm.targetNamespaces']
```

Comment:
Is there an advantage of using the OLM specific convention over the universal `fieldPath: metadata.namespace`?

Reply:
The target namespace may not be the same as the namespace in which the operator is installed.

Reply:
As discussed, most of the time the target namespace will be the same as the operator installation namespace. Unless the `operatorGroup` is manually modified with another namespace whose existence has to be guaranteed by the user.

Though I think I'm fine with the flexibility and with the concept of the rolebinding ensured by OLM in the target namespaces.